run/shared/species_database.yml

```diff
@@ -136,6 +136,55 @@ ALK4:
   Is_Advected: true
   Is_Gas: true
   MW_g: 58.12
+aoa:
+  FullName: Age of air uniform source tracer
+  Is_Advected: true
+  Is_Gas: true
+  Is_Tracer: true
+  Snk_Horiz: all
+  Snk_Mode: constant
+  Snk_Value: 0
+  Snk_Vert: surface
+  Src_Add: true
+  Src_Horiz: all
+  Src_Mode: constant
+  Src_Units: timestep
+  Src_Value: 1
+  Src_Vert: all
+  Units: days
```

Comment:
I wonder if we should call this something more specific, such as Lifetime_Units, or something. It gets confusing that there can be "Units" for a species in the species database that we will only use for specific species, and that will never be used for concentrations.

Reply:
The newly added `Units` field is meant to be used for all species, not just TransportTracers, though we currently only specify it for age of air species. The purpose is to specify the default units for the species. Where all existing species in GEOS-Chem are currently in units mol/mol dry air (and saved out as such in the SpeciesConc and Restart History collections), the aoa* species are in units days. I would eventually like to modify the diagnostic code to use this field to output the correct units for each species - as it currently stands the output files say that the aoa species are also in units mol/mol dry, but that is not correct.

Also, with the upcoming [modifications to the unit conversion code](https://github.com/geoschem/geos-chem/pull/1796), we would ideally skip converting units of species that have different units and/or no molecular weight defined.  For example, it doesn't make sense to convert aoa or the PassiveTracer to molec/cm2 at any time.